### PR TITLE
fix: Node.js 16 deprecation warning

### PR DESCRIPTION
**Pull request by devCrossNet**
I have no experience with GitHub action development. I just went through the code and updated all node version references to 20 and ran the `npm run all` script. Let me know if I need to change anything.

### CODE REVIEW

**Comment by brianfeister** [comment]
@AngelOnFira would we be able to get this merged? Would love to be on Node20 for all github actions

File: .gitignore [hunk 1]
```diff
@@ -106,3 +106,6 @@ lib/**/*
 .yarn/sdks
 .yarn/versions
 .yarn/install-state.gz
+
+# IDE
+.idea
```
Comment (AngelOnFira) [inline .gitignore:111]:
```suggestion
.yarn/install-state.gz
```

**Comment by AngelOnFira** [comment]
Yup, let's get this sorted 👍

**Comment by AngelOnFira** [comment]
Just for posterity, I ran CI in this PR after rebasing on main: https://github.com/SpicyPizza/create-envfile/pull/175

**Comment by AngelOnFira** [comment]
Thanks for the PR!

**Pull request merged**